<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interface</class>
 <widget class="QWidget" name="interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat Client</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_userlist">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>211</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_msgPage">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>36</y>
     <width>521</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_one">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>410</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>one</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_all">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>410</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>all</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Chatting with:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="chatuser">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>404</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_msg"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_send">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_file">
      <property name="text">
       <string>File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
